ref(debug-files): Split into smaller pages (#2878)
getsentry/sentry-docs: src/platforms/common/data-management/debug-files/upload/index.mdx

---
title: Uploading Files
sidebar_order: 3
description: "Learn about uploading debug information files to Sentry."
---

The most straightforward way to provide Sentry with debug information file is
to upload them using `sentry-cli`. Depending on your workflow, you may want to
upload as part of your build pipeline or when deploying and publishing your
application:

![](debug-files-workflow.png)

Files can be uploaded using the `upload-dif` command. This command will scan a
given folder recursively for files and upload them to Sentry:

<PlatformContent includePath="upload-dif" />

For all available options and more information refer to [_Uploading Debug
Information_](/product/cli/dif/#uploading-files).

> Always ensure that debug files are uploaded before deploying or releasing your

Reviewer: ```suggestion
Debug files should be uploaded before deploying or releasing your
```

application so that crash reports can be processed. For manual testing, use the
`sentry-cli upload-dif --wait` before sending the first native crash or error
event.

If you upload a file that was previously reported as missing by Sentry in a crash
report, it may take up to one hour until Sentry starts using this file for new
crash reports. Existing events and issues are not processed again.

In Sentry, your uploaded files are associated with projects. You can view and

> manage uploads at _Project Settings > Debug Files_. This screen displays the

Reviewer: ```suggestion
manage uploads at **Project Settings > Debug Files**. This screen displays the
```
